config/routes.rb

```diff
@@ -3,7 +3,15 @@
 
   namespace :api do
     namespace :v1 do
-      resources :states, only: [:index, :show]
+      resources :states, only: [:index, :show] do
+        resources :parks, only: [:index, :show]
+      end
+    end
+  end
+  
+  namespace :api do
+    namespace :v1 do
+      resources :parks, only: [:index, :show]
```

Comment:
Please move this up, so that you only have one api/v1 block

Reply:
Done!